### PR TITLE
[update] Upgrade babel

### PR DESCRIPTION
**Pull request by nhunzaker**
Upgrades Babel to 5.0

http://babeljs.io/blog/2015/03/31/5.0.0/

Side note: Is `babelify` needed anymore?

### CODE REVIEW

File: gulpfile.js/config/webpack.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ module.exports = function(env) {
       loaders: [
         {
           test: /\.js$/,
-          loader: 'babel-loader?experimental',
+          loader: 'babel-loader?stage=1',
```
Comment (nhunzaker) [inline gulpfile.js/config/webpack.js]: http://babeljs.io/docs/usage/experimental/

**Comment by greypants** [comment]
@nhunzaker stage=1 still what you recommend?  I remember you saying something about it in slack the other day. If so, I'll merge.

**Comment by nhunzaker** [comment]
@greypants Stage 2 is the default. I've been using stage 1 because it includes `...spreads` for React projects

**Comment by greypants** [comment]
@nhunzaker sorry i forgot about this - looks like you deleted the branch?

**Comment by nhunzaker** [comment]
@greypants weird. That's definitely the case. One tick.

**Comment by nhunzaker** [comment]
See #141

**Pull request closed without merging**